DuckDB.NET.Data/DuckDBConnection.TableFunction.cs

```diff
@@ -117,11 +117,19 @@ public static unsafe void Bind(IntPtr info)
             parameters[i] = value;
         }
 
-        var tableFunctionData = functionInfo.Bind(parameters);
-
-        foreach (var parameter in parameters)
-        {
-            ((DuckDBValue)parameter).Dispose();
+        TableFunction tableFunctionData;
+        try {
+            tableFunctionData = functionInfo.Bind(parameters);
+        } catch (Exception ex) {
```

Comment:
Let's put the rest of the code inside try block too. Also, please format the code like the rest of the codebase.